thoth/common/openshift.py

```diff
@@ -1489,6 +1490,7 @@ def schedule_mi_workflow(
             "KNOWLEDGE_PATH": knowledge_path,
             "MI_THOTH": "1" if mi_used_for_thoth else "0",
             "MI_MERGE": "1" if mi_merge else "0",
+            "MI_MERGE_PATH": mi_merge_path,
```

Comment:
```suggestion
            "MI_MERGE_PATH": mi_merge_path or "",
```

I think we assign an empty string if a parameter is `None`. Is that OK here (I don't have mi context here)?

Reply:
Yes, it is okay